share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/AfuncW.py

```diff
@@ -0,0 +1,109 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2023 PIConGPU contributors
+Authors: Hannes Wolf
+License: GPLv3+
+
+Implements the assignment functions of order 0 to 3 as well as a function to
+calculate the current deposition vector.
+
+"""
+
+
+# functions for single values
+def NGP(x):
+    """Generates hat function with width 1 (symmetrical around x=0) -
+    assignement-function of order 0 for single values x.
+
+    Parameters:
+    x (float): x value

```

Comment:
This comment does not help. Something along the lines of
`x (float): distance between particle and evaluation point` would be more explanatory.

Same for the other assignment functions.

Reply:
thanks for the proposition. I replaced the bad description with it.